docs/01-01-overview.md

```diff
@@ -1,3 +1,3 @@
 # Kyma Environment Broker (KEB) overview
 
-Kyma Environment Broker is a component that allows you to provision Kyma as a Runtime on clusters provided by third-party providers. It uses the Runtime Provisioner's API to install Kyma on a given cluster.
+Kyma Environment Broker is a component that allows you to provision Kyma as a [SAP BTP, Kyma runtime](https://kyma-project.io/#/?id=kyma-and-sap-btp-kyma-runtime) on clusters provided by third-party providers. It uses the Runtime Provisioner's API to install Kyma on a given cluster.
```

Comment:
KEB uses Provisioner to create a cluster, then uses Reconciler and LM (the target architecture - only LM) to install Kyma there